### PR TITLE
Website docs chips

### PR DESCRIPTION
**Pull request by technabors**
Convert chips.sketch to chips.md. Notes:

- Removed existing content as it was stale and only images
- convert basic chips content
- Create hero image in the new 'zoom' style based on CTUI use case.
- used 2x pngs with `{:width: $var}` specification based on art-board size of each image.

Note: Still awaiting When to use and Best practices content. Hold off on review until this delivered.

### CODE REVIEW

File: _component_design/chips.md [hunk 1]
```diff
@@ -13,32 +13,87 @@ last-modified: 2017-08-17
```
Comment (nimbinatus) [inline _component_design/chips.md]: Update status and last-mod date

File: _component_design/chips.md [hunk 2]
```diff
@@ -13,32 +13,87 @@ last-modified: 2017-08-17
...
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Interactions
+
+Clicking a chip reveals additional related information in a card that overlays the chip. 
+
+For example: Click a user chip, allows the user to view email address, permissions, contact info, or other similar data.
```
Comment (nimbinatus) [inline _component_design/chips.md]: Change to a full sentence. Example:
```markdown
For example, clicking a user chip allows the user to view their email address...
```

File: _component_design/chips.md [hunk 3]
```diff
@@ -13,32 +13,87 @@ last-modified: 2017-08-17
...
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Default chip
+
+Default chip is a single circle that contains a preview of the item the chip represents. This can be either letters to indicate the name, an image to indicate a user, or a product icon to indicate the type of device.
```
Comment (nimbinatus) [inline _component_design/chips.md]: Define `This` more clearly.

File: _component_design/chips.md [hunk 4]
```diff
@@ -13,32 +13,87 @@ last-modified: 2017-08-17
...
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+- Chips are best used with indicating a list of items that may be part of a different workflow. For example, requestors on tickets can be added and removed or management tasks that can be performed by requestors. Devices also offer a good option for chip use. 
+
+
+### Best practices
+
+- Avoid using chips in multiply locations on apage.
```
Comment (nimbinatus) [inline _component_design/chips.md]: s/`multiply`/`multiple`

File: _component_design/chips.md [hunk 5]
```diff
@@ -13,32 +13,87 @@ last-modified: 2017-08-17
...
+
+### When to use
+
+- Chips are best used with indicating a list of items that may be part of a different workflow. For example, requestors on tickets can be added and removed or management tasks that can be performed by requestors. Devices also offer a good option for chip use. 
+
+
+### Best practices
+
+- Avoid using chips in multiply locations on apage. 
+- Make sure the user is well aware of what the chip is.
```
Comment (nimbinatus) [inline _component_design/chips.md]: What does this mean?

File: _component_design/chips.md [hunk 6]
```diff
@@ -13,32 +13,87 @@ last-modified: 2017-08-17
...
-{% column left:"hxCol" %}
-{% figure [caption:"Basic"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/chips.svg){:width="100%"}
-{% endfigure %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+- Chips are best used with indicating a list of items that may be part of a different workflow. For example, requestors on tickets can be added and removed or management tasks that can be performed by requestors. Devices also offer a good option for chip use.
```
Comment (nimbinatus) [inline _component_design/chips.md]: Since there's only one bullet, I'd remove it.

Comment (nimbinatus) [inline _component_design/chips.md]: Second sentence isn't really a sentence (the end phrase is a phrase instead of a full thought).

Comment (nimbinatus) [inline _component_design/chips.md]: When you say "Devices," do you mean mobile devices? Laptops are a device, too.

File: _component_design/chips.md [hunk 7]
```diff
@@ -13,32 +13,87 @@ last-modified: 2017-08-17
...
 <div class="hxRow">
-{% column left:"hxCol-4" %}
-{% figure [caption:"Hover"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/chip-hover.svg){:width="100%"}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Expanded chip
+
+Expanded chips should be used when additional context is needed to differentiate the chip from the other chips or when the additional information is required by the user.
```
Comment (nimbinatus) [inline _component_design/chips.md]: Change to active voice:
```markdown
Use expanded chips when the user needs additional context to differentiate the chip from other chips or when the user needs to provide additional information.
```

File: _component_design/chips.md (hunk 6 above)
Comment (technabors) [reply _component_design/chips.md]: removed.

Comment (technabors) [reply _component_design/chips.md]: truncated to, "Chips are best used with indicating a list of items that may be part of a different workflow. For example, adding or removing a list of requestors on a ticket."

Comment (technabors) [reply _component_design/chips.md]: I removed this sentence, see above note.

File: _component_design/chips.md (hunk 4 above)
Comment (technabors) [reply _component_design/chips.md]: updated.

File: _component_design/chips.md (hunk 2 above)
Comment (technabors) [reply _component_design/chips.md]: Updated to, "For example, clicking a user chip allows the user to view their email address, permissions, contact info, or other similar data. "

File: _component_design/chips.md (hunk 7 above)
Comment (technabors) [reply _component_design/chips.md]: updated.

File: _component_design/chips.md (hunk 5 above)
Comment (technabors) [reply _component_design/chips.md]: Lets try, "- Ensure the user can concretely identify the object represented by the chip."

Comment (technabors) [reply _component_design/chips.md]: s/object/item

File: _component_design/chips.md (hunk 3 above)
Comment (technabors) [reply _component_design/chips.md]: Does this read better, "The preview can consist of the the following; letters to indicate the name, an image to indicate a user, or a product icon to indicate the type of device."

File: _component_design/chips.md [hunk 8]
```diff
@@ -7,38 +7,93 @@ usage: >
...
 </div>
+
 <div class="hxRow">
-{% column left:"hxCol-4" %}
-{% figure [caption:"Hover"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/chip-hover.svg){:width="100%"}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Expanded chip
```
Comment (araiford) [inline _component_design/chips.md]: Remove the Expanded Chip and Interactions sections.  They are not a requirement for SDP, and it's a concept right now.  There's a Jira story for Ty to run a visual design pass on expanded chips.  after that we can add the content back in.

File: _component_design/chips.md [hunk 9]
```diff
@@ -7,38 +7,93 @@ usage: >
...
+### When to use
+
+Chips are best used with indicating a list of items that may be part of a different workflow. For example, adding or removing a list of requestors on a ticket.
+
+
+### Best practices
+
+- Avoid using chips in multiple locations on a page. 
+- Ensure the user can concretely identify the item represented by the chip.
+- Using the expanded version can help ensure the information needed by the user is visible.
```
Comment (araiford) [inline _component_design/chips.md]: rm this line

File: _component_design/chips.md (hunk 1 above)
Comment (technabors) [reply _component_design/chips.md]: updated.

File: _component_design/chips.md [hunk 10]
```diff
@@ -7,38 +7,93 @@ usage: >
...
-last-modified: 2017-08-17
+status: stable
+last-modified: 2017-11-16
 ---
 
 {% include toc.html %}
 
+## Introduction
+
 <div class="hxRow">
```
Comment (nimbinatus) [inline _component_design/chips.md]: Ah, sorry. With the landing of #328, you'll need to add the `markdown="1"` attribute to the row divs. Forgot about that one :)

File: _component_design/chips.md (hunk 8 above)
Comment (technabors) [reply _component_design/chips.md]: Ok, I will note that @lfielder is building this out right now, but it is still a concept. I'll comment these sections out for the time being until they have interactions and spec finalized.

File: _component_design/chips.md (hunk 10 above)
Comment (technabors) [reply _component_design/chips.md]: added.

File: _component_design/chips.md (hunk 9 above)
Comment (technabors) [reply _component_design/chips.md]: commented out to include later

**Pull request merged**